runestone/parsons/css/parsons.css

```diff
@@ -252,10 +252,28 @@
 	font-style: italic;
 }
 
-.label {
-	font-family: "Ariel";
-	font-size: 12px;
-	color: #232375;
-	position: absolute;
-	right: 5px;
+.block-label {
+	font-family: "Courier";
+	font-size: 10px;
+	color: #b6a6a6;
+	padding: .2em 0.0em .3em .6em;
+	vertical-align: top;
+	white-space: pre;
+}
+
+.labels {
```

Comment:
So you removed .label and changed it to .labels

But the element in the parsons problem still has a class of .label 

.label is not a good choice for this name as it is already used by bootstrap.  So you should pick a different name and update the generated parsons element to use the new class name as well.

Reply:
I did realize that .label was reserved so I switched it to .block-label. I use .labels to style the the div that holds all the labels for a given block. I don't see where I am still adding .label to an element. Could you show me?